hypothesis-python/src/hypothesis/core.py

```diff
@@ -1135,13 +1129,13 @@ def find(
         )
     specifier.validate()
 
-    last = [None]
+    last = []  # type: List[Ex]
```

Comment:
Why not `last: List[Ex] = []`?

Reply:
Because that syntax requires Python 3.6+, and we're keeping Python 3.5 support at least until we finish the cleanup from Python 2.

Reply:
Oh, ok then.